decidim/decidim
# Fix enable authorizations listing in admin panel

#### :tophat: What? Why?
When an admin is enabling just certain authorizations from the system panel, the list of Authorizations in the admin panel is not restricted just to those enabled. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #11726

#### Testing
1. Visit admin area http://localhost:3000/admin/authorization_workflows
2. See menu and listing 
3. Visit system admin and disable several authorizations ( ex leave enabled just the `Identity documents`)
4. Revisit admin page, and see : 
    1. Menu reflects correctly the enabled authorizations
    2. The listing displays all the authorizations 
5. Apply patch & refresh
6. Repeat step 4, and see
    1. Menu reflects correctly the enabled authorizations
    2. The listing displays only the enabled authorizations  ( this is the fix )
    
### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
### Before 
![image](https://github.com/decidim/decidim/assets/105683/1c0dead0-d48f-48e4-936d-8efe5ebd8842)

### After 
![image](https://github.com/decidim/decidim/assets/105683/b2cb4640-73d8-4e26-9f9d-45d6d16afd84)


:hearts: Thank you!
